### PR TITLE
[DEV-1356] - add dns record for CMS Strapi

### PR DESCRIPTION
**Pull request by maurodandrea**
add dns record for CMS Strapi
Depends on #636 #637 #678

#### List of Changes
add dns record for CMS Strapi

#### Motivation and Context
add dns record for CMS Strapi

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [X] My change not requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .infrastructure/10_dns.tf [hunk 1]
```diff
@@ -107,3 +107,33 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
   records = ["google-site-verification=Z94dFrXZD0YqP-r5BY5ODb4NsbQBAggTGRZM9fNtOj0"]
   ttl     = 3600
 }
+
+# Add DNS record for CMS Strapi
+module "records" {
```
Comment (kin0992) [inline .infrastructure/10_dns.tf:112]:
```suggestion
module "cms_dns_records" {
```
or, if you have a better name, feel free to propose it

File: .infrastructure/10_dns.tf [hunk 2]
```diff
@@ -108,6 +108,36 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
   ttl     = 3600
 }
 
+# Add DNS record for CMS Strapi
+module "cms_dns_records" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-route53.git//modules/records?ref=bc63328714550fd903d2574b263833c9ce1c867e" # v2.11.0"
+
+  zone_id = module.dns_zone_cms.route53_zone_zone_id[keys(var.dns_domain_name_cms)[0]]
```
Comment (kin0992) [inline .infrastructure/10_dns.tf:115]:
```suggestion
  zone_id = aws_route53_zone.dev_portal.id
```
you should use the `dev_portal` zone

File: .infrastructure/10_dns.tf [hunk 3]
```diff
@@ -108,6 +108,36 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
...
+      type = "A"
+      alias = {
+        name                   = module.cms_load_balancer.dns_name
+        zone_id                = module.cms_load_balancer.zone_id
+        evaluate_target_health = false
+      }
+    }
+  ]
+
+  depends_on = [module.dns_zone_cms]
```
Comment (kin0992) [inline .infrastructure/10_dns.tf:138]:
```suggestion
```
this is not required anymore

**Review by kin0992 — CHANGES_REQUESTED** [review]

**Review by datalek — APPROVED** [review]

**Review by kin0992 — APPROVED** [review]

File: .infrastructure/dns.tf [hunk 4]
```diff
@@ -108,3 +108,31 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
...
+
+# Add DNS record for CMS Strapi
+module "cms_dns_records" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-route53.git//modules/records?ref=bc63328714550fd903d2574b263833c9ce1c867e" # v2.11.0"
+
+  zone_id = aws_route53_zone.dev_portal.id
+
+  records = [
+    {
+      name = ""
```
Comment (datalek) [inline .infrastructure/dns.tf:120]:
```suggestion
      name = "cms"
```

File: .infrastructure/dns.tf [hunk 5]
```diff
@@ -108,3 +108,31 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
...
+      type = "A"
+      alias = {
+        name                   = module.cms_load_balancer.dns_name
+        zone_id                = module.cms_load_balancer.zone_id
+        evaluate_target_health = false
+      }
+    },
+    {
+      name = "www"
```
Comment (datalek) [inline .infrastructure/dns.tf:129]:
```suggestion
      name = "www.cms"
```

File: .infrastructure/dns.tf [hunk 6]
```diff
@@ -108,3 +108,31 @@ resource "aws_route53_record" "devportal_google_site_verification_txt" {
...
+      type = "A"
+      alias = {
+        name                   = module.cms_load_balancer.dns_name
+        zone_id                = module.cms_load_balancer.zone_id
+        evaluate_target_health = false
+      }
+    }
```
Comment (datalek) [inline .infrastructure/dns.tf:136]:
```suggestion
    }
```

**Review by datalek — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Review by datalek — APPROVED** [review]

**Pull request merged**